src/node/server/serverPluginCss.ts

```diff
@@ -34,7 +40,9 @@ export const cssPlugin: ServerPlugin = ({ root, app, watcher, resolver }) => {
         ctx.type = 'js'
         // we rewrite css with `?import` to a js module that inserts a style
         // tag linking to the actual raw url
-        ctx.body = codegenCss(id, css, modules)
+        ctx.body = config.hmr
+          ? codegenCss(id, css, modules)
+          : `export default ${JSON.stringify(modules || css)}`
```

Comment:
LMK if you'd prefer to add an `enableHMR: boolean = false` or `config: ServerConfig` argument to `codegenCss` instead.

Reply:
Hmm, I think I messed up this part. Looking into it... 🙃